### DISCUSSION TITLE
privacy expectations in <https://ruffle.rs/demo>

### DISCUSSION THREAD

**Discussion by ousia** [ruffle-rs/ruffle#15333] [Q&A] [answered]
Hi there,

for a project I’m working on (and other ones might benefit from your answer to my question), I would like to know whether a local animation dragged and dropped in https://ruffle.rs/demo enjoys total privacy (I mean, no data is visible to any other that the local user).

Just in case you wonder, animations may contain personal data, so I need to know before recommending the use of https://ruffle.rs/demo.

Many thanks for your help.

**Comment by Dinnerbone** [comment, marked as answer]
Hi there!

Nothing leaves your browser on ruffle.rs, we don't collect usage data or anything of the sort. It's a statically served html and some javascript, and it entirely runs within your browser. All saved storage and such is also local to your browser and not anywhere else.

If you'd like extra peace of mind, you may confirm this by opening the developer tools of your favourite browser and check the Network tab.

Note that we cannot extend the same guarantees to the content (swf) itself. It may of course call some api or website, but it's the content doing that and not Ruffle or our demo.

Hope this helps!

**Reply by ousia** [reply]
Many thanks for your reply, @Dinnerbone.

This perfectly fits my expectations.
